lib/Dancer2/Core/App.pm

```diff
@@ -34,6 +34,44 @@ with qw<
 
 sub supported_engines { [ qw<logger serializer session template> ] }
 
+sub with_plugins {
```

Comment:
[ I also suggest to have a method for loading plugins instead of hiding that in an Moo attribute.]

Something like this?